<?xml version="1.0" ?>
<launch>
  <arg name="object"   default="Suchobjekt" />
  <arg name="model"    default="$(arg object)" />

  <!--Start Learning-->
  <node pkg="learn_object" type="learnKeypoints" name="learn_keypoints" output="screen" >
    <param name="object_name"               type="str"    value="$(arg object)" /> 
    <param name="model_filename"            type="str"    value="$(arg model)" /> 
    <param name="feature"                   type="str"    value="SURF" />
    <param name="path"                      type="str"    value="$(find learn_object)/model/"/> 
    <param name="binMode"                   type="bool"   value="true" /> 
    <param name="saveKeypointsGlobal"       type="bool"   value="true" />  
    <param name="TrainImageNumber"          type="int"    value="12" />
  </node>
</launch>
